janusgraph-cql/src/test/java/org/janusgraph/JanusGraphCassandraContainer.java

```diff
@@ -97,7 +98,7 @@ public JanusGraphCassandraContainer() {
     }
 
     public JanusGraphCassandraContainer(boolean bindDefaultPort) {
-        super(getCassandraImage() + ":" + getVersion());
+        super(DockerImageName.parse(getCassandraImage() + ":" + getVersion()).asCompatibleSubstituteFor(DEFAULT_IMAGE));
```

Comment:
Good idea.

I think it would be better to open an issue in the testcontainers repo.

Reply:
From reading at thier repo seems like intended change for some safety.

Even that building that on names only, seems like a limitation with no much gain. Maybe they had to many issued open from people wrongly configuring it.

I'll open an issue, but for current situation it's seems reasonable enough to follow thier recommendation.

Reply:
@farodin91 you still think an issue on `testcontainers` is needed ?

Reply:
I think creating an issue would be good but i can do this.